pkg/auth/authz_handler.go

```diff
@@ -16,6 +16,12 @@ const (
 	amsActionCreate          string = "create"
 	capabilityName           string = "bare_metal_installer_admin"
 	capabilityType           string = "Account"
+
+	// FakePayloadUsername for AuthAgentAuth return value
+	FakePayloadUsername string = "jdoe123@example.com"
```

Comment:
Why not something more clear like admin@example.com?

Reply:
just use the one from: https://github.com/openshift/assisted-service/blob/9ca9af7cddc07b0b101e37a5ad818dbec3f1fefe/tools/auth_keys_generator.go#L39
can change to admin :)